changelogs/fragments/5124-compatibility-virtualmedia-resource-location.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - redfish - for compatibility due to Redfish spec changes the virtualMedia resource location from Manager to System (https://github.com/ansible-collections/community.general/pull/5124).
```

Comment:
This PR also adds new features. That is completely missing in this changelog fragment.

Reply:
Thank you for your suggestion, I will update the changelog fragment.

Reply:
New features must be listed in the `minor_changes` category.